tests/sqllogictests/suites/base/03_common/03_0026_insert_into_tuple.test

```diff
@@ -50,7 +50,7 @@ select t:a, t:b, t:c, t:d, t:e, t:f from t2
 0 -10 -0.9 y 2022-10-01 2022-10-01 12:00:00.000000
 
 statement ok
-CREATE TABLE IF NOT EXISTS t3(id Int, t Tuple(a Tuple(m Int64, n Int64), b Tuple(x Int64, y Int64))) Engine = Fuse
+CREATE TABLE IF NOT EXISTS t3(id Int, t Tuple(a Tuple(m Int64, _n Int64), b Tuple("X" Int64, "y@#z" Int64))) Engine = Fuse
```

Comment:
If field name is Chinese?

Reply:
yes, Chinese is also support as special characters in quotes.